source/conf.py

```diff
@@ -344,3 +344,37 @@
 # If true, figures, tables and code-blocks are automatically numbered if they
 # have a caption. At same time, the numref role is enabled.
 numfig = True
+
+# -- Custom configuration -------------------------------------------------
+
+from jinja2 import Environment, StrictUndefined
+
+# Use Jinja to parse RST files first
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    jinja_env = Environment(undefined=StrictUndefined)
```

Comment:
This should work. I didn't test it though.

```python
jinja_env = Environment(undefined=StrictUndefined, variable_start_string='{{{{', variable_end_string='}}}}')
```

I can't imagine how that would affect anything outside of this function.